html/elements/datalist.json

```diff
@@ -39,7 +39,7 @@
               "version_added": true
             },
             "webview_android": {
-              "version_added": false
+              "version_added": true
```

Comment:
```suggestion
              "version_added": "4.4.3"
```

Reply:
Great! I was unsure what version to suggest.